### PR TITLE
EZP-27539: As an Editor I want to add a translation in the Translation view tab

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZP-27539
Requires: https://github.com/ezsystems/repository-forms/pull/182

### CODE REVIEW

File: src/lib/Form/Type/Content/Translation/TranslationAddType.php [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class TranslationAddType extends AbstractType
+{
+    /** @var LanguageService */
+    protected $langaugeService;
```
Comment (mikadamczyk) [inline src/lib/Form/Type/Content/Translation/TranslationAddType.php]: There is typo in `$langaugeService`

**Comment by m-tyrala** [comment]
There's also a problem with fields names - while adding translation all fields names are changed to "Value".

File: src/lib/Form/Type/Content/Translation/TranslationAddType.php (hunk 1 above)
Comment (webhdx) [reply src/lib/Form/Type/Content/Translation/TranslationAddType.php]: Whoops. Thanks :)

**Comment by webhdx** [comment]
@m-tyrala Fixed.

**Comment by lserwatka** [comment]
@dew326 @webhdx you will have to re-base

**Comment by webhdx** [comment]
@lserwatka Rebased. @m-tyrala Can we prioritize this one? Content translation feels like an important feature ;)

**Pull request merged**